schema/GenomeAnnotation.mcf

```diff
@@ -117,7 +123,81 @@ name: "genomeAssemblyYear"
 typeOf: schema:Property
 rangeIncludes: schema:Number
 domainIncludes: dcs:GenomeAssembly
-description:"The year in which the genome assembly was released."
+description: "The year in which the genome assembly was released."
+
+Node: dcid:genomeReferenceConsortiumAssemblyName
+name: "genomeReferenceConsortiumAssemblyName"
+typeOf: schema:Property
+rangeIncludes: schema:Text
+domainIncludes: dcs:GenomeAssembly
+description: "The Genome Refrence Consortium name for the genome assembly."
+
+Node: dcid:genBankAccession
+name: "genBankAssemblyAccession"
+typeOf: schema:Property
+rangeIncludes: schema:Text
+domainIncludes: dcs:GenomeAssembly,dcs:GenomeAssemblyUnit,dcs:Chromosome
+description: "The accession version of the GenBank assembly or sequence element."
+
+Node: dcid:refSeqAccession
+name: "refSeqAssemblyAccession"
+typeOf: schema:Property
+rangeIncludes: schema:Text
+domainIncludes: dcs:GenomeAssembly,dcs:GenomeAssemblyUnit,dcs:Chromosome
+description: "The accession version of the RefSeq assembly or sequence element."
+
+Node: dcid:ncbiBioProject
+name: "ncbiBioProject"
+typeOf: schema:Property
+rangeIncludes: schema:Text
+domainIncludes: dcs:GenomeAssembly
+description: "A BioProject is a collection of biological data related to a single initiative originating from a single organization or from a consortium. A BioProject record provides users a single place to find links to the diverse data generated for that project and deposited into the archival databases maintained by members of the INSDC. Typical examples of a BioProject include a multiisolate project for sequencing multiple strains of a bacterial species, or a monoisolate project for the genome and transcriptome of a particular organism."
```

Comment:
BioProject is a NCBI specific entity. Should I drop the ncbi prefix?

Reply:
@shanth what do you think?

Reply:
Since it is specific to ncbi, I feel its okay to include it in the name.  Maybe in the description say... "An NCBI (...expand NCBI...) BioProject is a collection of ..."

?